drp.py

```diff
@@ -256,6 +264,22 @@ def handle_error(exc, retry=True):
         sublime.set_timeout_async(connect_background, 0)
 
 
+def get_git_url(window):
+    for folder in window.folders():
+        gitcfg_path = folder+"/.git/config"
+
+        if (os.path.exists(gitcfg_path)):
+            f = open(gitcfg_path, "r")
```

Comment:
If you're not going to use the CLI method to retrieve the remote url, you should close this file handle after you don't need it anymore, ideally using a `with` statement.